client/src/features/ProjectPageV2/ProjectPageHeader/ProjectTemplateInfoBanner.tsx

```diff
@@ -0,0 +1,161 @@
+/*!
+ * Copyright 2024 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+import cx from "classnames";
+import { useCallback, useState } from "react";
+import { Button, ListGroup, ModalBody, ModalHeader } from "reactstrap";
+import { Diagram3Fill } from "react-bootstrap-icons";
+
+import PrimaryAlert from "../../../components/PrimaryAlert";
+import ScrollableModal from "../../../components/modal/ScrollableModal";
+
+import PermissionsGuard from "../../permissionsV2/PermissionsGuard";
+import type { Project } from "../../projectsV2/api/projectV2.api";
+import ProjectShortHandDisplay from "../../projectsV2/show/ProjectShortHandDisplay";
+import { useGetProjectsByProjectIdCopiesQuery } from "../../projectsV2/api/projectV2.api";
+import { useGetUserQuery } from "../../usersV2/api/users.api";
+
+import useProjectPermissions from "../utils/useProjectPermissions.hook";
+import styles from "./ProjectTemplateInfoBanner.module.css";
+
+interface ProjectCopyListModalProps {
+  copies: Project[];
+  isOpen: boolean;
+  project: Project;
+  title: string;
+  toggle: () => void;
+}
+
+export function ProjectCopyListModal({
+  copies,
+  isOpen,
+  project,
+  title,
+  toggle,
+}: ProjectCopyListModalProps) {
+  return (
+    <ScrollableModal
+      data-cy="copy-list-modal"
+      backdrop="static"
+      isOpen={isOpen}
+      toggle={toggle}
+      size="lg"
+      centered
+    >
+      <ModalHeader toggle={toggle}>
+        <span className="fw-normal">{title} </span>
+        {project.namespace}/{project.slug}
+      </ModalHeader>
+      <ModalBody className={cx("overflow-y-scroll", styles.modalBody)}>
```

Comment:
```suggestion
      <ModalBody className={styles.modalBody}>
```

The scrollbar should appear only if necessary (screenshots taken when `overflow-y-scroll` is removed):
![Screenshot 2025-01-06 at 11 56 01](https://github.com/user-attachments/assets/c80e019c-2863-47ca-9662-8324146c0c53)
![image](https://github.com/user-attachments/assets/15ec896c-43e0-446a-8bd5-18857f8e5466)

Reply:
Fixed in 44455c28e59968780314cc8c2bcddaa7c33649ac